### PR TITLE
Radio c tests

### PR DESCRIPTION
**Pull request by MarekPorwisz**

### CODE REVIEW

**Comment by MarekPorwisz** [comment]
Each tests needs to have description

**Comment by MarekPorwisz** [comment]
Add missing license

File: tests/subsys/openthread/radio_stub.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* needed here so the static device_get_binding does not get renamed */
```
Comment (lmaciejonczyk) [inline tests/subsys/openthread/radio_stub.c:7]: Missing copyright section

File: tests/subsys/openthread/radio_test.c [hunk 2]
```diff
@@ -0,0 +1,753 @@
+#include <ztest.h>
```
Comment (lmaciejonczyk) [inline tests/subsys/openthread/radio_test.c:7]: Missing copyright section

**Review by lmaciejonczyk — CHANGES_REQUESTED** [review]
Generally looks ok from my perspective.
One thing you should especially take care of is code formatting which does not fullfile general requirements in many places. I strongly advise using clang-formatter or uncrustify. Zephyr already provides configuration files for this tools.

**Pull request closed without merging**